documentation/synthetic_tokens/explainer.md

```diff
@@ -109,6 +99,4 @@ After the expiration timestamp for the synthetic tokens, anyone may settle the c
 
 After this value is returned to the contract and the contract is settled, any token holder can redeem the tokens against the contract. Redemption of the tokens returns the token holder collateral equal to the price identifier value returned by the UMA DVM.  <!-- TODO: add link to glossary for “price identifier” --> 
 
-<p align="center">
-  <img src="st_expiration.png" align="center" width="500">
-</p>
+![](st_expiration.png)
```

Comment:
nit: this image looks a little small IMO (it has a lot of content).

Totally up to you whether to change, it was just the only one that looked a little out of place.

Reply:
let's change when we figure out the final solution for the images.